html-css/override-extend-variables/step4-instructions.md

```diff
@@ -132,19 +132,17 @@ PatternFly is an open source design system created to enable consistency and usa
   &lt;/div&gt;
 &lt;/div&gt;
 </pre>
-
-2) <strong>Create a wrapper around the Cards.</strong> Multiple Cards were just added and we need to create a wrapper around them. 
-
-Add an opening `<div>` tag to the first line, and a closing `</div>` tag to the bottom of the file.
-
-(3) <strong>Add a layout class.</strong> 
-
-To the opening `<div>` tag, add the gallery layout class: `pf-l-gallery`.
-
+ 
+3) <strong>Create a wrapper around the Cards.</strong> Multiple Cards were just added and you need to create a wrapper around them. 
+<strong>Add an opening</strong> `<div>` tag to the first line, and a closing `</div>` tag to the bottom of the file.
```

Comment:
bold the whole first sentence

Reply:
3) <strong>Create a wrapper around the Cards.</strong> Multiple Cards were just added and you need to create a wrapper around them. 
<strong>Add an opening</strong> `<div>` tag to the first line, and a closing `</div>` tag to the bottom of the file.
sb
3) <strong>Create a wrapper around the Cards. Multiple cards were just added and you need to create a wrapper around them. </strong>
Add an opening `<div>` tag to the first line, and a closing `</div>` tag to the bottom of the file.